azure-pipelines.yml

```diff
@@ -0,0 +1,115 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+jobs:
+
+- job: Linux
+  
+  strategy:
+    matrix:
+      Python35:
+        python.version: '3.5'
+      Python36:
+        python.version: '3.6'
+      Python37:
+        python.version: '3.7'
+      Python38:
+        python.version: '3.8'
+  
+  pool:
+    vmImage: 'ubuntu-latest'
+  
+  steps:
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '$(python.version)'
+        addToPath: true
+        architecture: 'x64'
+      displayName: 'Use Python $(python.version)'
+
+    - script: |
+        python -m pip install --upgrade pip
+        python -m pip install setuptool
+        python -m pip install wheel
+        pip install --only-binary=numpy,scipy numpy scipy
+        pip install cython
+        pip install .
+      displayName: 'Installing scikit-multiflow'
+
+    - script: |
+        pip install pytest
+        pip install pytest-cov
+        pytest tests --showlocals -v --cov=skmultiflow
+      displayName: 'Running tests'
+
+- job: macOS
+  
+  strategy:
+    matrix:
+      Python37:
+        python.version: '3.7'
+      Python38:
+        python.version: '3.8'
+  
+  pool:
+    vmImage: 'macOS-latest'
+  
+  steps:
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '$(python.version)'
+        addToPath: true
+        architecture: 'x64'
+      displayName: 'Use Python $(python.version)'
+
+    - script: |
+        python -m pip install --upgrade pip
+        python -m pip install setuptool
+        python -m pip install wheel
+        pip install --only-binary=numpy,scipy numpy scipy
+        pip install cython
+        pip install .
+      displayName: 'Installing scikit-multiflow'
+
+    - script: |
+        pip install pytest
+        pip install pytest-cov
+        pytest tests --showlocals -v --cov=skmultiflow
+      displayName: 'Running tests'
```

Comment:
I'd create a `requirements-dev.txt` for `pytest`, `wheel`, and others. Then I install both `requirements.txt` and `requirements-dev.txt` in this stage.  
So changes will be made only in one place.

Reply:
I like the simplicity of this approach, thanks for the suggestion. I will try it.

Reply:
Ended up with a hybrid approach. The following steps must be run individually:

```
python -m pip install --upgrade pip
python -m pip install setuptool
```

Reply:
It's clean. Thanks @jacobmontiel